### PR TITLE
FW-2353 If both msg and image is sent from dashboard, text is not coming in widget

### PR DESCRIPTION
**Pull request by ChannuBS**
### What do you want to achieve?
-If both msg and image is sent from the dashboard, the text is not coming in the widget

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

- add any attachment with a text message in the dashboard and chat widget side and send them
- we can receive the attachment data with text message 
Screenshot 2021-06-29 at 10 48 59 AM

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/css/app/mck-sidebox-1.0.css [hunk 1]
```diff
@@ -3051,6 +3051,17 @@ display: none;
 .mck-group-inner .mck-msg-left .mck-msg-box:after {
 	border-right-color: #e6e5ec;
 }
+.message-class-left {
```
Comment (devashishmamgain) [inline webplugin/css/app/mck-sidebox-1.0.css:3054]: use some other naming, remove class and also see alternative for left
reason is that in RTL what comes on left will be on right side, and this naming might confuse us in future.

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 2]
```diff
@@ -8258,7 +8258,7 @@ var userOverride = {
                 var messageClass = 'vis';
                 var progressMeterClass = 'n-vis';
                 var attachmentBox = 'n-vis';
-
+                var kmAttchMsg= ''
```
Comment (devashishmamgain) [inline webplugin/js/app/mck-sidebox-1.0.js:8261]: add ;

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 3]
```diff
@@ -8357,10 +8357,26 @@ var userOverride = {
...
+                        messageClass = 'vis km-attach-msg-right';
+                        attachmentBox = 'km-attach-msg-right';
+                        if (!msg.message) {
+                            messageClass = 'n-vis';
+                        }
+                    }
+                } else {
+                    msgBoxColorStyle =
+                        'km-custom-widget-background-color-secondary';
+                    if (kmAttchMsg === 'km-attach-msg') {
```
Comment (devashishmamgain) [inline webplugin/js/app/mck-sidebox-1.0.js:8372]: code inside the if else looks same, figure out a better way to implement it with minimal code and without repeating

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 4]
```diff
@@ -8258,7 +8258,7 @@ var userOverride = {
                 var messageClass = 'vis';
                 var progressMeterClass = 'n-vis';
                 var attachmentBox = 'n-vis';
-
+                var kmAttchMsg= '';
```
Comment (devashishmamgain) [inline webplugin/js/app/mck-sidebox-1.0.js:8261]: add space before =

**Review by devashishmamgain — DISMISSED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Review by Ojashwa — COMMENTED** [review]
- Getting this type of UI after scrolling in the chat window.
---
https://user-images.githubusercontent.com/22856546/123832897-73834e80-d923-11eb-8141-c7cb68ea8a82.mov

**Review by Ojashwa — APPROVED** [review]

**Review by Prochnost55 — APPROVED** [review]

**Pull request merged**